packages/app-page-builder/src/admin/views/PageBlocks/BlocksByCategoriesDataList.tsx

```diff
@@ -114,7 +114,7 @@ const BlocksByCategoriesDataList = ({
         listQuery?.data?.pageBuilder?.listBlockCategories?.data || [];
     const pageBlocksData: PbPageBlock[] = listQuery?.data?.pageBuilder?.listPageBlocks?.data || [];
 
-    const [blocksList, blockCategoriesList] = useCategoriesListData(
+    const [blocksList, filteredBlockCategoriesList] = useCategoriesListData(
```

Comment:
I know it's a nitpick, but since you've applied the `blockCategoriesList` => `filteredBlockCategoriesList` (thanks!), let's be consistent and also apply the same to `blocksList` and `useCategoriesListData`. Unless I'm missing something, both of these represent filtered data.